docs/docs/add-a-manifest-file.md

```diff
@@ -1,32 +1,32 @@
 ---
-title: Adding a Manifest File
+title: Hinzufügen einer Manifest Datei
 ---
 
-If you've run an [audit with Lighthouse](/docs/audit-with-lighthouse/), you may have noticed a lackluster score in the "Progressive Web App" category. Let's address how you can improve that score.
+Wenn du einen [Test mit Lighthouse](/docs/audit-with-lighthouse/) ausführst, hast du vielleicht schon die fehlende Bewertung in der Kategorie "Progressive Web App" gesehen. Hier erklären wir dir, wie du diese verbessern kannst.
 
-But first, what exactly _are_ PWAs?
+Aber zuerst, was _sind_ überhaupt PWAs?
 
-They are regular websites that take advantage of modern browser functionality to augment the web experience with app-like features and benefits. Check out [Google's overview](https://developers.google.com/web/progressive-web-apps/) of what defines a PWA experience and the [Progressive web apps (PWAs) doc](/docs/progressive-web-app/) to learn how a Gatsby site is a progressive web app.
+PWAs sind normale Webseiten die sich Funktionen moderner Browser zunutze machen um das User Erlebnis so zu verändern, dass die Webseite sich wie eine App anfühlt. Hier kannst du [Google's Übersicht (in Englisch)](https://developers.google.com/web/progressive-web-apps/) lesen, auf der erklärt wird, was die Merkmale und Vorteile von PWAs sind. Hier findest du außerdem unser [Progressive web apps (PWAs) Dokument](/docs/progressive-web-app/), mit dem wir dir zeigen, wie deine Gatsby Seite eine PWA werden kann.
 
-The inclusion of a web app manifest is one of the three generally accepted [baseline requirements for a PWA](https://alistapart.com/article/yes-that-web-project-should-be-a-pwa#section1).
+Das Bereitstellen eines Web App Manifests ist eine der drei [grundlegenden Anforderungen an eine PWA](https://alistapart.com/article/yes-that-web-project-should-be-a-pwa#section1).
 
-Quoting [Google](https://developers.google.com/web/fundamentals/web-app-manifest/):
+Zitat [Google](https://developers.google.com/web/fundamentals/web-app-manifest/):
```

Comment:
Maybe we should add that it is a translated quote?

Reply:
Yeah, we could, but we haven't done it in the other translations. I don't really know if there is any value in that and think we should just keep it consistent.

So maybe looking through all the translations and unifying these things is a task for later?

Reply:
I don't think it is necessary for a direct translation since the content doesn't change with the language.